node/Makefile

```diff
@@ -0,0 +1,21 @@
+SHELL=/usr/bin/env bash
+
+LOTUS_PATH:=extern/lotus
+
+GOCC?=go
+
+submodule:
+	git submodule set-branch -b feat/nv18-fevm node/extern/lotus
+	git submodule sync
+	git submodule update --init --recursive --remote
+	go mod tidy
```

Comment:
Maybe we could get away without cloning lotus into a submodule, and simply using a replace directive in the node go.mod to point to the latest commit in the branch. However, this is fine too.

Reply:
We may not be able to get away with replace directives in go.mod because the lotus dependencies (e.g. FFI) have to be built.